pkg/controllers/execution/execution_controller.go

```diff
@@ -206,7 +206,7 @@ func (c *Controller) syncToClusters(clusterName string, work *workv1alpha1.Work)
 			}
 		} else {
 			err = c.tryCreateWorkload(clusterName, workload)
-			if err != nil {
+			if err != nil && !apierrors.IsAlreadyExists(err) {
```

Comment:
at line pkg/controllers/execution/execution_controller.go:189, if there are two Manifests(A and B).
1. creating the work for the first time, create A succeed, B failed, so the applied will be false.
2. create the work for the second time, create A failed(already exists), and create B succeed.
3. Then create the work continually, A will be failed always(always an error of already exists).

@RainbowMango